pkg/daemon/controller/controller.go

```diff
@@ -67,13 +67,37 @@ type ServiceController struct {
 	watchNamespace string
 }
 
+// Bootstrap starts all Windows services marked as necessary for node bootstrapping as defined in the given data
+func (sc *ServiceController) Bootstrap(desiredVersion string) error {
+	var cm core.ConfigMap
+	err := sc.client.Get(sc.ctx,
+		client.ObjectKey{Namespace: sc.watchNamespace, Name: servicescm.NamePrefix + desiredVersion}, &cm)
+	if err != nil {
+		return err
+	}
+	cmData, err := servicescm.Parse(cm.Data)
+	if err != nil {
+		return err
+	}
+
+	bootstrapSvcs := []servicescm.Service{}
+	for _, svc := range cmData.Services {
+		if !svc.Bootstrap {
+			// services are pre-sorted by priority, with all bootstrap services ordered towards the front of the slice
+			break
+		}
+		bootstrapSvcs = append(bootstrapSvcs, svc)
+	}
```

Comment:
can break this out into a new function for cleaner code
`func getBootstrapServices(data) []servicescm.Service{}`

Reply:
I feel like getBootstrapServices() is better suited as a function on the `data` object (servicescm package). I'll add this in